packages/acceptance-tests/pkg-tests-specs/sources/commands/add.test.js

```diff
@@ -116,5 +117,50 @@ describe(`Commands`, () => {
         });
       }),
     );
+
+    test(`it should clean the cache when cache lives inside the project`, makeTemporaryEnv({
```

Comment:
I'm still unsure if `add` should be the place for these tests.

Reply:
I think it's fine 👍